src/utils/berths.ts

```diff
@@ -1,30 +1,56 @@
 import { List } from 'immutable';
-import { get } from 'lodash';
+import get from 'lodash/get';
 
 import { Berths } from '../components/berths/types';
-import { SelectedServices } from '../types/services';
+import {
+  SelectedServices,
+  SelectedServicesProps,
+  SelectedWinterServices,
+  SelectedWinterServicesProps
+} from '../types/services';
 
 import { BerthType } from '../types/berth';
 import { WinterStorageType } from '../types/winterStorage';
 
 import { BoatTypesBerthsQuery_harbors } from './__generated__/BoatTypesBerthsQuery';
 import { WinterAreasQuery_winterStorageAreas } from './__generated__/WinterAreasQuery';
 
-export const getBerthFilterByValues = (values: {}, selectedServices: SelectedServices) => {
+export const getBerthFilterByValues = (
+  values: {},
+  selectedServices: SelectedServices | SelectedWinterServices
+) => {
   const width = Number(get(values, 'boatWidth', '').replace(',', '.')) * 100;
   const length = Number(get(values, 'boatLength', '').replace(',', '.')) * 100;
```

Comment:
Should using an util here as well

Reply:
Since this conversion is only happening inside a utility function and nowhere else, I'd rather keep it here than extract it to another utility function which will add another layer of abstraction.